gradle-jdks/src/main/java/com/palantir/gradle/jdks/JdksExtension.java

```diff
@@ -31,6 +32,8 @@ public abstract class JdksExtension {
 
     protected abstract MapProperty<JavaLanguageVersion, JdkExtension> getJdks();
 
+    public abstract MapProperty<String, File> getCaCerts();
```

Comment:
Might be cleaner to take PEM formatted CA certs as strings, that way we don't need to think about file I/O. iirc they can be fed to keytool via stdin when no `-file` is specified.